bin/release_tag.rb

```diff
@@ -19,6 +20,7 @@
 post_review = StringIO.new
 
 ManageIQ::Release.repos_for(opts).each do |repo|
+  next if !opts[:skip].nil? && opts[:skip].include?(repo.name)
```

Comment:
Since it's marked as `:strings`, I think it will always be an Array, but even so, this might be cleaner:

```suggestion
  next if Array(opts[:skip]).include?(repo.name)
```